ignite/services/network/network.go

```diff
@@ -100,14 +113,25 @@ func CollectEvents(ev events.Bus) Option {
 
 // New creates a Builder.
 func New(cosmos CosmosClient, account cosmosaccount.Account, options ...Option) Network {
+	cosmos.Context().InterfaceRegistry.RegisterImplementations(
```

Comment:
We should avoid changing context like this, can we register light client types by default in cosmos client, is there any implications? 

Fyi, this line failing the unit tests. - nil pointer

Reply:
this query is too specific, we will use it only one time, I think we don't need to inject in `cosmoclient` by default, we can create a copy of the context to use into the `node` object, WDYT?